store/s3.go

```diff
@@ -12,39 +13,39 @@ import (
 	"github.com/google/uuid"
 )
 
-type S3Store struct {
+type AWSS3Storage struct {
 	bucket          string
 	region          string
-	accessKey       string
+	accessKeyID     string
 	secretAccessKey string
 }
 
```

Comment:
Please make sure the `AWSS3Storage` implements the `Storage` interface by adding 
`var _ Storage = (*AWSS3Storage)(nil)`

Reply:
Yeah. That would be good. This is a tricky Go practice: https://splice.com/blog/golang-verify-type-implements-interface-compile-time/. Although this is not a mandatory practice, many people do this for additional safety.

Reply:
Thanks! Is it okay to add `var _ Storage = &AWSStorage{}`?

Reply:
Fixed in [2a8bdff](https://github.com/medibloc/panacea-data-market-validator/pull/71/commits/2a8bdff83a6e7c3a2dd76cc419386b626e665868).